Jarvis-Platform/jarvis-studio
# Feature/dedicated mixins

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Add dedicated mixin for each type of collection / doc.


## Todos
- [x] Create all different mixins
- [x] Define properties in each mixins


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/dedicated-mixins
```

1. Try to navigate between all `data-workflows` pages, it should run properly.


## Impacted Areas in Application
- `src/mixins/data-workflows/*`
- `src/views/data-workflows/*`
